cmd/migrate_from_milvus.go

```diff
@@ -0,0 +1,404 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/milvus-io/milvus/client/v2/column"
+	"github.com/milvus-io/milvus/client/v2/entity"
+	"github.com/milvus-io/milvus/client/v2/milvusclient"
+	"github.com/pterm/pterm"
+
+	"github.com/qdrant/go-client/qdrant"
+
+	"github.com/qdrant/migration/pkg/commons"
+)
+
+type MigrateFromMilvusCmd struct {
+	SourceUrl                      string `help:"Source Milvus URL, e.g. https://your-milvus-hostname" required:"true"`
```

Comment:
These config flags are improved upon in https://github.com/qdrant/migration/pull/26.
With better re-usability and clarity.